src/actions/signupActions.js

```diff
@@ -0,0 +1,14 @@
+import { Post } from '../utilities/apiRequests';
+
+const signupUser = userDetails => () => Post('/auth/signup', userDetails)
+  .then(response => response).catch((error) => {
+    if (error.message === 'Network Error') {
```

Comment:
What happens when other errors occur? How will they be handled?

Reply:
They are passed to the method, in the container component (SignupPage.jsx), that calls the action creator.